mongodb/plan.sh

```diff
@@ -21,11 +21,11 @@ pkg_build_deps=(
 pkg_bin_dirs=(bin)
 pkg_lib_dirs=(lib)
 pkg_include_dirs=(include)
-pkg_svc_run="mongod --config $pkg_svc_config_path/mongod.conf"
 pkg_svc_user=hab
 pkg_svc_group=hab
 pkg_exports=(
   [port]=mongod.net.port
+  [address]=sys.ip
```

Comment:
You can only export configuration defined within `default.toml` so this won't work. However, you definitely do have access to this from a bind. You can simply ask for `sys.ip` just like you'd ask for `cfg.port` - there's no need to export it